test/features/email/data/repository/calendar_event_repository_impl_test.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:core/data/model/source_type/data_source_type.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:jmap_dart_client/jmap/account_id.dart';
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:jmap_dart_client/jmap/mail/calendar/properties/event_id.dart';
+import 'package:jmap_dart_client/jmap/mail/calendar/reply/calendar_event_accept_response.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:tmail_ui_user/features/email/data/datasource/calendar_event_datasource.dart';
+import 'package:tmail_ui_user/features/email/data/repository/calendar_event_repository_impl.dart';
+import 'package:tmail_ui_user/features/email/domain/exceptions/calendar_event_exceptions.dart';
+
+import 'calendar_event_repository_impl_test.mocks.dart';
+
+@GenerateNiceMocks([MockSpec<CalendarEventDataSource>()])
+void main() {
+  final calendarEventNetworkDataSource = MockCalendarEventDataSource();
+  final calendarEventDataSource = {
+    DataSourceType.network: calendarEventNetworkDataSource};
+  final calendarEventRepository = CalendarEventRepositoryImpl(calendarEventDataSource);
+  final accountId = AccountId(Id('123'));
+  final blobId = Id('blobId');
+
+  group('calendar event repository', () {
```

Comment:
```suggestion
  group('calendar event repository test', () {
```

Reply:
Why should this be changed? Currently if we run the test, we will have these in log:
- `calendar event repository should return response when data source return response`
- `calendar event repository should throw exception when data source throw exception`
If this change is made, these will be printed in log instead:
- `calendar event repository test should return response when data source return response`
- `calendar event repository test should throw exception when data source throw exception`
Does the later make sense more?

Reply:
If you are a reader, do you understand what it is before running? We have to let everyone understand what it is, not just having to run it to know what it is?

Reply:
Done in:
- https://github.com/linagora/tmail-flutter/pull/2801/